docs/api/crossplaneio/crossplane/stacks-crossplane-io-v1alpha1.md

```diff
@@ -178,6 +220,95 @@ Name | Type | Description
 
 
 
+## ResourceEngineConfiguration
+
+ResourceEngineConfiguration represents a configuration for a resource engine, such as helm2 or kustomize.
+
+Appears in:
+
+* [HookConfiguration](#HookConfiguration)
+* [StackConfigurationBehavior](#StackConfigurationBehavior)
+* [StackConfigurationBehaviors](#StackConfigurationBehaviors)
+
+
+Name | Type | Description
+-----|------|------------
+`type` | string | 
+
+
+
+## StackConfigurationBehavior
+
+StackConfigurationBehavior specifies an individual behavior, by listing resources which should be processed.
+
+Appears in:
+
+* [StackConfigurationBehaviors](#StackConfigurationBehaviors)
+
+
+Name | Type | Description
+-----|------|------------
+`hooks` | [map[string]github.com/crossplaneio/crossplane/apis/stacks/v1alpha1.HookConfigurations](#HookConfigurations) | The key for Hooks is an event name which represents the lifecycle event that the controller should respond to. There are certain events that are recognized. Currently, only &#34;reoncile&#34; is recognized.
+`engine` | [ResourceEngineConfiguration](#ResourceEngineConfiguration) | ResourceEngineConfiguration represents a configuration for a resource engine, such as helm2 or kustomize.
+
+
+
+## StackConfigurationBehaviors
+
+StackConfigurationBehaviors specifies behaviors for the stack
+
+Appears in:
+
+* [StackConfigurationSpec](#StackConfigurationSpec)
+
+
+Name | Type | Description
+-----|------|------------
+`crds` | [map[github.com/crossplaneio/crossplane/apis/stacks/v1alpha1.GVK]github.com/crossplaneio/crossplane/apis/stacks/v1alpha1.StackConfigurationBehavior](#StackConfigurationBehavior) | 
```

Comment:
Is this right? It seems like it should be links to the `GVK` alias above and to `StackConfigurationBehavior`. I'm noticing that the GVK alias also doesn't have any `Appears in:`, and I'm noticing that the map in `StackConfigurationBehavior.hooks` also looks funny.

Reply:
Looks like `crossdocs` doesn't know how to handle a map of other structs :disappointed: options here are:
1. Just go with this current state and we can assume that experimental docs are okay for experimental features.
2. Exclude these docs for the time being (issue here is that when we incorporate automatically generating the docs there will be a diff until this is fixed, but we could also just cross that bridge when we come to it).
3. Fix crossdocs, which would maybe delay this release (not recommended).

Reply:
my vote would be #1, keep the docs as is so that the new type is at least included (all alpha types should be included), but the docs have a small content bug in them.  I'm OK with that for now.